Pod/Classes/AnimationSugar.swift

```diff
@@ -104,3 +171,124 @@ public class Animation {
+     */
+    func rotate(angle angle: CGFloat) {
+        self.transform = CGAffineTransformRotate(self.transform, angle)
+    }
+    
+    /**
+     Translate the view by altering its `frame` property.
+     
+     - Parameter tx: Translation distance along the x axis
+     - Parameter ty: Translation distance along the y axis
+     */
+    func translateFrame(tx tx: CGFloat, ty: CGFloat) {
+        var frame = self.frame
+        frame.origin.x = frame.origin.x + tx
+        frame.origin.y = frame.origin.y + ty
+        self.frame = frame
+    }
+    
+    /**
+     Stretch or shring the view using its `frame` property.
+     
+     - Parameter deltaWidth: The amount to stretch the frame in the x direction
+     - Parameter deltaHeight: The amount to stretch the frame in the y direction
+     */
+    func stretchFrame(deltaWidth deltaWidth: CGFloat, deltaHeight: CGFloat) {
+        var frame = self.frame
+        frame.size.width = frame.size.width + deltaWidth
+        frame.size.height = frame.size.height + deltaHeight
+        self.frame = frame
+    }
+    
+    /**
+     Resize the view's `frame` property.
+     
+     - Parameter width: The new width
+     - Parameter height: The new height
+     */
+    func resizeFrame(width width: CGFloat, height: CGFloat) {
+        var frame = self.frame
+        frame.size.width = width
+        frame.size.height = height
+        self.frame = frame
+    }
+    
+    /**
+     Flip the view 180 degrees horizontally (along the y axis)
+     
+     - Parameter perspectiveDistance: The simulated distance between the observer
+     and the view, in points. Since the flip is in three dimensions, a perspective
+     transform is applied. Smaller values will have an exaggerated perspective.
+     Default value is 1000 points.
+     */
+    func flipHorizontal(perspectiveDistance: CGFloat = 1000.0) {
+        self.layer.transform.m34 = -1.0 / perspectiveDistance
+        self.layer.transform = CATransform3DRotate(self.layer.transform, CGFloat(M_PI), 0, 1.0, 0)
+    }
+    
+    /**
+     Flip the view 180 degrees vertically (along the x axis)
+     
+     - Parameter perspectiveDistance: The simulated distance between the observer
+     and the view, in points. Since the flip is in three dimensions, a perspective
+     transform is applied. Smaller values will have an exaggerated perspective.
+     Default value is 1000 points.
+     */
+    func flipVertical(perspectiveDistance: CGFloat = 1000.0) {
+        self.layer.transform.m34 = -1.0 / perspectiveDistance
+        self.layer.transform = CATransform3DRotate(self.layer.transform, CGFloat(M_PI), 1.0, 0, 0)
+    }
+    
+    /**
+     Fade in a view by animating its `alpha`.
+     
+     - Parameter duration: The duration of the animation
+     - Returns: An `Animation` that can be chained or altered.
+     */
+    func fadeIn(duration duration: NSTimeInterval = Animation.defaultAnimationDuration) -> Animation {
+        self.alpha = 0.0
```

Comment:
Would it be worth calling `.hidden = false` here? Paul and I have been using a similar function that also turns off hidden. Might be a side-effect depending on how you think of it.